towhee/functional/data_collection.py

```diff
@@ -512,14 +493,14 @@ def inner():
                     if raw:
                         yield buff
                     else:
-                        yield DataCollection.unstream(buff if isinstance(buff, list) else [buff])
+                        yield buff if isinstance(buff, list) else [buff]
```

Comment:
we expect the user can also use DC for the batch. For example, apply a towhee operator.

Reply:
When creating the dag, each batch would be considered its own node if we make them DCs. Wouldnt it be better to just return the batches as a list, then act on those lists? Or are you expecting people to use batch to split the data collection into something like train, validate, test and perform further actions on those batches.

Reply:
instead of returning a dc[dc, dc, dc, ...] we return a dc[list, list, list, ..].